content/numpy/concepts/math-methods/terms/degrees/degrees.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: '.degrees()'
+Description: 'Converts radians to degrees.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Arrays'
+  - 'Functions'
+  - 'NumPy'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In NumPy, the **`.degrees()`** function converts an angle measure from radians to degrees.
+
+## Syntax
+
+```pseudo
+numpy.degrees(x)
+
+numpy.degrees([x, x, x])
```

Comment:
```suggestion
numpy.degrees(x, out=None, where=True)
```

Reply:
This is the syntax of the `.degrees()` method. Including the parameters that are used very often